scripts/release_scripts/update_changelog_and_credits_test.py

```diff
@@ -33,31 +33,55 @@
 from scripts import common
 from scripts.release_scripts import update_changelog_and_credits
 
-import github  # isort:skip pylint: disable=wrong-import-position
+from typing import List, Union
+from typing_extensions import Final
 
-RELEASE_TEST_DIR = os.path.join('core', 'tests', 'release_sources', '')
+import github  # isort:skip pylint: disable=wrong-import-position
 
-MOCK_RELEASE_SUMMARY_FILEPATH = os.path.join(
+RELEASE_TEST_DIR: Final = os.path.join('core', 'tests', 'release_sources', '')
+MOCK_RELEASE_SUMMARY_FILEPATH: Final = os.path.join(
     RELEASE_TEST_DIR, 'release_summary.md')
 
-MOCK_CHANGELOG_FILEPATH = os.path.join(RELEASE_TEST_DIR, 'CHANGELOG')
-MOCK_AUTHORS_FILEPATH = os.path.join(RELEASE_TEST_DIR, 'AUTHORS')
-MOCK_CONTRIBUTORS_FILEPATH = os.path.join(RELEASE_TEST_DIR, 'CONTRIBUTORS')
-MOCK_ABOUT_PAGE_CONSTANTS_FILEPATH = 'about_temp_file.ts'
-MOCK_PACKAGE_JSON_PATH = os.path.join(RELEASE_TEST_DIR, 'mock_package.json')
-MOCK_FECONF_PATH = os.path.join(RELEASE_TEST_DIR, 'feconf.txt')
-
-MOCK_UPDATED_CHANGELOG_FILEPATH = os.path.join(
+MOCK_CHANGELOG_FILEPATH: Final = os.path.join(RELEASE_TEST_DIR, 'CHANGELOG')
+MOCK_AUTHORS_FILEPATH: Final = os.path.join(RELEASE_TEST_DIR, 'AUTHORS')
+MOCK_CONTRIBUTORS_FILEPATH: Final = os.path.join(
+    RELEASE_TEST_DIR, 'CONTRIBUTORS'
+)
+MOCK_ABOUT_PAGE_CONSTANTS_FILEPATH: Final = 'about_temp_file.ts'
+MOCK_PACKAGE_JSON_PATH: Final = os.path.join(
+    RELEASE_TEST_DIR, 'mock_package.json'
+)
+MOCK_FECONF_PATH: Final = os.path.join(RELEASE_TEST_DIR, 'feconf.txt')
+
+MOCK_UPDATED_CHANGELOG_FILEPATH: Final = os.path.join(
     RELEASE_TEST_DIR, 'UPDATED_CHANGELOG')
-MOCK_UPDATED_CHANGELOG_FILEPATH_FOR_REMOVAL_TEST = os.path.join(
+MOCK_UPDATED_CHANGELOG_FILEPATH_FOR_REMOVAL_TEST: Final = os.path.join(
     RELEASE_TEST_DIR, 'UPDATED_CHANGELOG_FOR_REMOVAL_TEST')
-MOCK_UPDATED_AUTHORS_FILEPATH = os.path.join(
+MOCK_UPDATED_AUTHORS_FILEPATH: Final = os.path.join(
     RELEASE_TEST_DIR, 'UPDATED_AUTHORS')
-MOCK_UPDATED_CONTRIBUTORS_FILEPATH = os.path.join(
+MOCK_UPDATED_CONTRIBUTORS_FILEPATH: Final = os.path.join(
     RELEASE_TEST_DIR, 'UPDATED_CONTRIBUTORS')
 
-
-def read_from_file(filepath):
+# Here we use MyPy ignore because the pool_size argument is required by
+# Requester.__init__(), but it is missing from the typing definition in
+# Requester.pyi. We therefore disable type checking here. Here is the
+# type definition:
+# https://github.com/PyGithub/PyGithub/blob/001970d4a828017f704f6744a5775b4207a6523c/github/Requester.pyi#L97
```

Comment:
Can we report this as an issue?

Reply:
Reported: https://github.com/PyGithub/PyGithub/issues/2322